modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -37,15 +36,30 @@ The update should be atomic; that is, both the `containerImage` and `version` fi
 
 . Terminate any workload using the kernel module on the node being upgraded.
 
-. Remove the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label on the node. This unloads the kernel module from the node.
+. Remove the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label on the node.
+This unloads the kernel module from the node:
++
+[source,terminal]
+----
+$ oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>-
+----
 
-. If required, perform any additional maintenance required on the node for the kernel module upgrade.
+. If required, as the cluster administrator, perform any additional maintenance required on the node for the kernel module upgrade.
++
+If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label value to the new `$moduleVersion` as set in the `Module`.
 
 . Add the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>=$moduleVersion` label to the node. `$moduleVersion` should be equal to the new value of the `version` field in the `Module` CR.
```

Comment:
```suggestion
. Add the `kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>=$moduleVersion` label to the node. `$moduleVersion` should be equal to the new value of the `version` field in the `Module` CR.
```

Reply:
Made these changes and added "Run the following command to add..."